cli/cli.py

```diff
@@ -63,9 +63,12 @@ def notify_on_crash(func):
     def decorator(*args, **kwargs):
         try:
             func(*args, **kwargs)
+
         except KeyboardInterrupt:
-            pass
-        else:
+            log.info("Caught ctrl-c; Exiting...")
```

Comment:
Can you explain this one to me? The interrupt handler above this function already states when it catches the interrupt, so why state this again?

Reply:
I honestly missed the interrupt handler above, because I'm in the habit of just catching the exception myself. I put the statement their when I was trying to figure out why I got a crash message for `ctrl-c` before i figure out it was the `else:` instead of `except:` and never took it out because I never saw it.

Reply:
Removed the log statement and replaced it with `pass` again